<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReplayWindow</class>
 <widget class="QMainWindow" name="ReplayWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1115</width>
    <height>785</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ReplayWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="infoLayout">
      <item>
       <widget class="QLabel" name="alertName">
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
	color: red;
	margin: 5px;	
	margin-left: 20px
}</string>
        </property>
        <property name="text">
         <string>TAK</string>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="MonitorGraphicsView" name="graphicsView">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="timelineLayout">
      <item>
       <widget class="QPushButton" name="playButton">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../Resources/resources.qrc">
          <normaloff>:/icons_64/play.png</normaloff>:/icons_64/play.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="videoSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="acceptButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Zatwierd≈∫</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MonitorGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header location="global">MonitorGraphicsView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>playButton</sender>
   <signal>clicked()</signal>
   <receiver>ReplayWindow</receiver>
   <slot>onPlay()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>acceptButton</sender>
   <signal>clicked()</signal>
   <receiver>ReplayWindow</receiver>
   <slot>onAccept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1067</x>
     <y>754</y>
    </hint>
    <hint type="destinationlabel">
     <x>557</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onPlay()</slot>
  <slot>onSliderValueChanged(int)</slot>
  <slot>onAccept()</slot>
 </slots>
</ui>
